python/llm/example/GPU/LangChain/vllm.py

```diff
@@ -0,0 +1,45 @@
+from ipex_llm.langchain.vllm.vllm import VLLM
```

Comment:
change to `from ipex_llm.langchain.llms import VLLM`

Reply:
Using `from ipex_llm.langchain.llms import VLLM` will encounter dependency problems.

Codes in the llm [folder](https://github.com/intel-analytics/ipex-llm/tree/main/python/llm/src/ipex_llm/langchain/llms) will require langchain version `0.0.184`, which is quite old.

If we put vllm related code in llms folder, then the imports in __init__.py will encounter errors: 
```python

from typing import Dict, Type
from langchain.llms.base import BaseLLM

from .bigdlllm import *
from .transformersllm import TransformersLLM
from .transformerspipelinellm import TransformersPipelineLLM
from .vllm import VLLM, VLLMOpenAI

__all__ = [
    "BigdlNativeLLM",
    "LlamaLLM",
    "BloomLLM",
    "GptneoxLLM",
    "ChatGLMLLM",
    "StarcoderLLM",
    "TransformersLLM",
    "TransformersPipelineLLM",
    "VLLM",
    "VLLMOpenAI"
]

type_to_cls_dict: Dict[str, Type[BaseLLM]] = {
    "BigdlNativeLLM": BigdlNativeLLM,
    "LlamaLLM": LlamaLLM,
    "BloomLLM": BloomLLM,
    "GptneoxLLM": GptneoxLLM,
    "ChatGLMLLM": ChatGLMLLM,
    "StarcoderLLM": StarcoderLLM,
    "TransformersPipelineLLM": TransformersPipelineLLM,
    "TransformersLLM": TransformersLLM,
    "VLLM": VLLM,
    "VLLMOpenAI": VLLMOpenAI
}
```

If we want to put vllm.py in llms, then we have the following options:
1. Upgrade langchain dependency from `0.0.184` to a higher version
2. Delete the imports in llms.__init__.py so that user import from a specific sub-file

Any comments on this?